### ISSUE TITLE
Add support for KNIME 4.0

### ISSUE THREAD

**Issue by danielesser** [SiLeBAT/FSK-Lab#340]
I am unable to install the BfR FSK-Lab nodes in KNIMe 4.0. It seems the nodes require a dependency, which is only available in KNIME 3.x versions. Dependency `org.knime.ext.r.type` is pinned to version rage `[3.0.0,4.0.0)`.

```
Cannot complete the install because one or more required items could not be found.
  Software being installed: BfR FSK-Lab 1.5.7.201906241421 (de.bund.bfr.knime.fsklab.feature.feature.group 1.5.7.201906241421)
  Missing requirement: BfR FSK-Lab 1.5.7.201906241421 (de.bund.bfr.knime.fsklab.feature.feature.group 1.5.7.201906241421) requires 'org.knime.ext.r.type [3.0.0,4.0.0)' but it could not be found
```

**Comment by mfilter** [comment]
Hi Daniel, thanks for pointing to this. We are working on fixing this asap.

**Comment by miguelalba** [comment]
Thanks Daniel. We just fixed it in #7ac6205d7268d742e8e7a37f450fa638adab8300 The range of the dependency was fixed to KNIME 3 in the feature. We still have to make a build on this.

**Comment by danielesser** [comment]
@miguelalba Awesome! Thank you! Looking forward to see the new release. Any idea when this is planned for?
